app/src/test/java/org/simple/clinic/instantsearch/InstantSearchUpdateTest.kt

```diff
@@ -324,10 +324,11 @@ class InstantSearchUpdateTest {
             gender = Gender.Male,
             age = null),
         address = OngoingNewPatientEntry.Address.BLANK.withColonyOrVillage(patientPrefillInfo.address),
-        identifier = identifier)
+        alternativeId = alternateId,
```

Comment:
Now we have a mix of these? Maybe change in a different PR later?

Reply:
I've changed this in this PR in a later commit.